src/screens/Login/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { isEmpty } from 'lodash';
+import validator from 'validator';
+
+import Alert from 'components/Alert';
+import Button from 'components/Button';
+import WarningIcon from 'components/Icon/WarningIcon';
+import Input from 'components/Input';
+import AuthLayout from 'components/Layout/Auth';
+
+const LoginScreen = (): JSX.Element => {
+  const { t } = useTranslation();
+  const [formInput, setformInput] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState<string[]>([]);
+  const [formLoading, setFormLoading] = useState(false);
+  const validateInput = (input: { email: string; password: string }) => {
+    const errors = [];
+    if (!input.email) {
+      errors.push(t('error.email_blank'));
+    }
+    if (!input.password) {
+      errors.push(t('error.password_blank'));
+    }
+    if (!validator.isEmail(input.email)) {
+      errors.push(t('error.email_invalid'));
+    }
+    setErrorMessage(errors);
+  };
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setFormLoading(true);
+    try {
+      validateInput(formInput);
+    } catch (error) {
+      console.log('error: ', error);
+    }
+    setFormLoading(false);
+  };
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setformInput({ ...formInput, [event.target.name]: event.target.value });
+  };
+
+  return (
+    <AuthLayout headerMessage={t('auth.heading')}>
```

Comment:
Could you recheck this form again?
It shows the wrong label for Email, and missing the padding for the input
![Challenge_-_Web_App_–_Figma](https://user-images.githubusercontent.com/7344405/218432621-d922f06b-a714-499d-8ecc-ada23b9cba1e.png)

Reply:
![React_App](https://user-images.githubusercontent.com/7344405/219543455-8efe3c80-f0d0-4949-a81f-46e377329bbb.png)

It seems the email text is incorrect